poleno/utils/models.py

```diff
@@ -115,6 +115,26 @@ def __init__(self, *args):
         self._choices = choices
         self._inverse = inverse
 
+class OriginalValuesMixin(object):
+    u"""
+    Mixin to keep track of original values of changed fields.
+    """
+    tracked_fields = []
+
+    def __init__(self, *args, **kwargs):
+        super(OriginalValuesMixin, self).__init__(*args, **kwargs)
+        self._tracked_fields = {}
+        for name in self.tracked_fields:
+            self._tracked_fields[name] = getattr(self, name)
+
+    def get_value(self, name):
```

Comment:
@viliambalaz Este som zabudol. Pls nazvi to `get_original_value`, kedze `get_value` skor dava pocit, ze to bude aktualna value.

Reply:
:ok: